docs/manual/library/games/faq.md

```diff
@@ -0,0 +1,22 @@
+## Frequently Asked Questions (FAQ)
+---------------------
+
+### I can't find a game in my library, why?
+
+See [here](../../gettingStarted/helpAndTroubleshooting/troubleshooting.md#my-games-are-gone-or-missing).
+
+### I uninstalled a library plugin but its games are still there, why?
```

Comment:
This or all page about removing games should mention that Playnite never automatically removes games from the library. Not even when an integration is removed.

Reply:
https://github.com/JosefNemec/PlayniteDocs/pull/1/commits/435b76721dfac21da166d33b4aee6d17c5925611